module/item/systems/repair.js

```diff
@@ -274,25 +339,29 @@ class RepairData {
     }
 
     get repairDC() {
-        return this.diagram.system.craftingDC - repairModifier + this.enchantsDC
+        return this.diagram.system.craftingDC + repairModifier + this.enchantsDC
     }
 
-    get repairDCForumulae() {
-        let formulae = `${this.diagram.system.craftingDC}[${game.i18n.localize('WITCHER.Repair.params.craftingDC')}] - ${repairModifier}[${game.i18n.localize('WITCHER.Repair.params.repairMod')}]`
+    get repairDCForumula() {
+        let formula = `${this.diagram.system.craftingDC}[${game.i18n.localize('WITCHER.Repair.params.craftingDC')}] - ${Math.abs(repairModifier)}[${game.i18n.localize('WITCHER.Repair.params.repairMod')}]`
         const echantsCount = this.enchantsCount
         if (echantsCount) {
-            formulae += ` + ${echantsCount} * ${perEnchantModifier}[${game.i18n.localize('WITCHER.Repair.params.enchants')}]`
+            formula += ` + ${echantsCount} * ${perEnchantModifier}[${game.i18n.localize('WITCHER.Repair.params.enchants')}]`
         }
 
-        return formulae
+        return formula
     }
 
     get enchantsDC() {
         return this.enchantsCount * perEnchantModifier
     }
 
     get skillName() {
-        return game.i18n.localize(this.executor.system.skills.cra.crafting.label).replace(" (2)", "")
+        return game.i18n.localize(WITCHER.skillMap.crafting.label)
```

Comment:
The config should be loaded from CONFIG.WITCHER... so that modules or world scripts can modify stuff

Reply:
This is ineed the `CONFIG.WITCHER` or am I loading it incorrectly?
`import { WITCHER } from "../../setup/config.js";`

Reply:
You are loading the system defined WITCHER config. In the setup this is added to the global CONFIG object where modules or world scripts can change things.

Reply:
![grafik](https://github.com/user-attachments/assets/4a4d154f-cb07-4ce0-a364-ef6a78453c11)

Reply:
This was fixed in one of the latest commits.
Forgot to resolve it, sorry